pkg/interactive/securitygroups/security_groups.go

```diff
@@ -77,13 +77,18 @@ func GetSecurityGroupIds(r *rosa.Runtime, cmd *cobra.Command,
 	return securityGroupIds
 }
 
-func isValidSecurityGroup(sg types.SecurityGroup) bool {
+func isValidSecurityGroup(sg types.SecurityGroup, id string) bool {
 	if aws.Ec2ResourceHasTag(sg.Tags, tags.RedHatManaged, strconv.FormatBool(true)) {
 		return false
 	}
 	if awsSdk.ToString(sg.GroupName) == "default" {
 		return false
 	}
+	if id != "" {
+		if aws.Ec2ResourceHasTag(sg.Tags, fmt.Sprintf("kubernetes.io/cluster/%s", id), "owned") {
```

Comment:
Hello~
I'm Just wondering if we can get away with checking substring "kubernetes.io/cluster/" with value "owned" instead of having to pass in the cluster id.

I am assuming this is required for displaying list of additional sec. group options in interactive mode?
As long as the security group is not owned by any cluster, then would it be safe to display for selection?

Reply:
I agree with this approach.
Currently, we check the tag with the cluster/infra ID in the backend.
If we decide to check the substring, changes should be applied both to CLI and backend, for consistency.